src/crimson/net/ProtocolV2.cc

```diff
@@ -732,7 +732,6 @@ seastar::future<bool> ProtocolV2::client_connect()
           peer_global_seq = server_ident.global_seq();
 
           // TODO: lossless policy
-          ceph_assert(server_ident.flags() & CEPH_MSG_CONNECT_LOSSY);
```

Comment:
this is but a workaround, which helped the test move a step further. but the test still fails like https://pastebin.com/tPUncSz1